src/IssueCreator/Main.cs

```diff
@@ -338,24 +354,17 @@ private void LstSelectedTags_MouseDoubleClick(object sender, MouseEventArgs e)
         {
             MoveItem(lstSelectedTags, lstAvailableTags);
         }
-
-        private void FrmMain_FormClosing(object sender, FormClosingEventArgs e)
-        {
-            s_settings.Serialize(SettingsFile);
-        }
         #endregion
 
-        private void BtnRefreshEpics_Click(object sender, EventArgs e)
+        private async void BtnRefreshEpics_Click(object sender, EventArgs e)
```

Comment:
Should return a `Task` or `ValueTask` instead of `async void`. I would expect the UI to do something (spinning mouse, for e.g.) that ends when this task is done without hanging the UI.

Reply:
Changed to Task.

Reply:
Because this is an event handler for an UI element, it cannot have Task as the return type. This is a special case around async design to make event handlers work.